sechub-pds-solutions/tern/tests/run-suite.sh

```diff
@@ -34,7 +34,7 @@ then
     do
         create_build_folder
         run_test "sechub-test-$test_case" "${test_cases[$test_case]}"
-        remove_build_folder 
+        #remove_build_folder 
```

Comment:
Is this intentionally commented out?

Reply:
Yes, because otherwise my results will be deleted as well.